### DISCUSSION TITLE
Welcome to SDK-Doc Discussions!

### DISCUSSION THREAD

**Discussion by AnthonyYates** [SuperOffice/SDK-Doc#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

As SuperOffice docs are in the middle of a transition at the moment, this Discussions are is just temporary. But feel free to ask questions and give feedback!

**Comment by josteink** [comment]
This is a new shiny tool, and we'll see how well it works in practice.

First of all though, it might be nice to let people know that it exists, otherwise we probably won't get much feedback.

Should we add it to the repo's [README](https://github.com/SuperOffice/SDK-Doc/blob/master/README.md)?

**Reply by xt1** [reply]
It's still under development.
